src/components/results/securityanalysis/security-analysis-result-n.tsx

```diff
@@ -58,11 +72,38 @@ export const SecurityAnalysisResultN: FunctionComponent<
         [intl, sortProps, filterProps, filterEnums]
     );
 
+    const csvHeaders = useMemo(
+        () => columnDefs.map((cDef) => cDef.headerName),
+        [columnDefs]
+    );
```

Comment:
useMemo from useMemo?  
maybe doing it as one call?
```suggestion
    const [csvHeaders, columnDefs] = useMemo(() => {
        let columnDefs = securityAnalysisTableNColumnsDefinition(intl, sortProps, filterProps, filterEnums);
        return [columnDefs, columnDefs.map((cDef) => cDef.headerName)];
    }, [intl, sortProps, filterProps, filterEnums]);
```